src/nisarqa/utils/metadata_checks.py

```diff
@@ -166,9 +167,9 @@ def verify_metadata_cubes(
     product: nisarqa.NisarProduct, fail_if_all_nan: bool = True
 ) -> None:
     """
-    Verify the input product's coordinate grid metadata cubes are valid.
+    Verify the input product's coordinate grid metadata cube LUTs are valid.
```

Comment:
Let's just say "cubes" here -- later on we say that "Coordinate grid metadata cubes are the 3D LUTs [...]".

```suggestion
    Verify the input product's coordinate grid metadata cubes are valid.
```